src/types/domains.ts

```diff
@@ -1,57 +1,20 @@
 import { DomainColor } from '~gql';
 
-export enum Color {
```

Comment:
I think we don't need two separate enums (and a mapping between them) and we can simply use the GraphQL enum directly

Reply:
Works for me.